provider/app.py

```diff
@@ -19,25 +19,35 @@
 from consumer import Consumer
 from database import Database
 from urlparse import urlparse
+import requests
+
 
 app = Flask(__name__)
 database = Database()
-
 consumers = dict()
 
-
 @app.route('/triggers/<namespace>/<trigger>', methods=['PUT'])
 def postTrigger(namespace, trigger):
     body = request.get_json(force=True, silent=True)
-
     triggerFQN = '/' + namespace + '/' + trigger
-    logging.info("About to create consumer with " +
-                 triggerFQN + " " + str(body))
-    createAndRunConsumer(triggerFQN, body)
-    logging.info("Finished creating consumer.")
-    result = {'success': True}
-    return jsonify(result)
+    logging.info("Ensuring user has access rights to post a trigger")
+    response = requests.get(body["triggerURL"])
+    status_code = response.status_code
+    logging.info("Repsonse status code {}".format(status_code))
```

Comment:
More specific please: 

```python
"[{}] Status code when authorizing trigger: {}".format(triggerFQN, status_code)
```
Remember  that anyone looking at these messages is trying to figure out what happened or what went wrong in a particular scenario. Also keep in mind that logging messages from (potentially) many different threads will be interleaved. We need to provide as much context as possible in each message.